docs/extensions/readme.md

```diff
@@ -350,4 +352,58 @@ LongRunningOperationRetryTimeout | `int` | Retry timeout in seconds for Long Run
     "AcceptLanguage": "EN-GB",
     "LongRunningOperationRetryTimeout": 9001
 }
-```
\ No newline at end of file
+```
+
+## x-ps-client-side-filter
+Defines client-side filters
+
+**Parent element**: `x-ps-cmdlet-info Object`
+
+**Schema**:
+
+Field Name | Type | Description
+---|:---:|---
+serverSideResultCommand | `string` | Cmdlet to run to get server-side results. Use "." to indicate the current cmdlet.
+serverSideResultParameterSet | `string` | Parameter set to get server-side results.
+clientSideParameterSet | `string` | Parameter set to get parameters from.
+filters | `x-ps-client-side-filter-definition[]` | Filters
+
+**Examples**:
+```json5
+{
+    "serverSideResultCommand": ".",
+    "serverSideResultParameterSet": "BatchAccount_List",
+    "clientSideParameterSet": "BatchAccount_Get",
+    "filters": ...
+}
+```
+
+## x-ps-client-side-filter-definition
+Defines a single client-side filter
+
+**Parent element**: `x-ps-client-side-filter Object`
+
+**Schema**:
+
+Field Name | Type | Description
+---|:---:|---
+type | `string` | Type of filter. Supported: 'wildcard', 'logicalOperation'

```

Comment:
>logicalOperation [](start = 58, length = 16)

What is the purpose of logicalOperation type?

Reply:
80f233f